news-sentiment/README.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Analyze the Sentiment of News Headlines
+description: A Google Sheet with Apps Script that Analyzes the sentiment of news headlines retrieved for a user specified topic from a free News API.
+labels: Apps Script, Sheets, Cloud Natural Language
+material_icon: Feed
+create_time: 2021-03-22
+update_time: 2021-03-22
+---
+
+This solution consists of a Google Sheet that uses Apps Script to fetch the current news headlines from the free news API `newsapi.org` using a user provded `topic`.
+
+Once the news headlines are fetched and loaded onto the sheet, it runs a Natural Language sentiment detection for each headline. 
+As the sentiments are returned from the Google Cloud Language API, the sheet is updated with both a numeric score as well as a `happy`, `meh` or `sad` icon depending on the sentiment.
+ 
+## Technology highlights
+
+- The sheet provides an extra menu option: **News Headlines Sentiments** and when clicked renders the option: **Analyze News headlines...**.  When selected, it calls the `showNewsPrompt()` function and triggers a popup dialog to allow the user to enter their own news topic of choice. 
+- After entering a topic, the function `analyzeNewsHeadlines()` executes and calls the following key functions:
+   * `reformatSheet()` Clears and reformats the sheet.
+   * `getHeadlinesArray()` Fetches a fresh set of headlines by passing the user entered topic to the News API. It then syncs the returned headlines to the sheet.
+   * `getSentiments()` Iterates through each headline and calls the Natural Language API to obtain a sentiment value and then updates the sheet with values.
+
+## Try it
+
+### Pre-Steps - Obtaining your API keys
+To run this solution you will need to obtain 2 API keys, one from the [Google Cloud Natural Language API](https://cloud.google.com/natural-language), and the second from the free News API @ http://newsapi.org/.
```

Comment:
Make news API a link

Reply:
Done. I left the actual URL in the link text.